ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersMixedRolesError.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+      <td
+        class="py-1"
+        :class="{ 'border-error-left': showBorderError }"
+        :colspan="4"
+        data-test-id="invalid-group-mixed-owners"
+        >
+          <div
+          class="error-text my-6 text-center"
+          :data-test-id="`mixed-owners-msg-group-${groupId}`"
+          >
+            <span>
+            {{ mixedRoleErrorMsg }}
+            </span>
+          </div>
+      </td>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, reactive, toRefs } from '@vue/composition-api'
+import { useGetters } from 'vuex-composition-helpers'
+import { MixedRolesErrors } from '@/resources'
+
+export default defineComponent({
+  name: 'HomeOwnersMixedRolesError',
+  props: {
+    groupId: {
+      type: Number,
+      required: true
+    },
+    showBorderError: {
+      type: Boolean,
+      required: true
+    }
+  },
+  setup () {
+    const {
+      getMhrRegistrationHomeOwnerGroups
+    } = useGetters<any>([
+      'getMhrRegistrationHomeOwnerGroups'
+    ])
+
+    const localState = reactive({
+      mixedRoleErrorMsg: computed(() => getMhrRegistrationHomeOwnerGroups.value.length === 1
```

Comment:
No action required here just wanted to highlight a small thing. 
I'm sure you may already be familiar but typescript 3.7 and beyond you can leverage optional chaining as a means to protect against null or undefined values. 
ie: `getMhrRegistrationHomeOwnerGroups.value?.length...`

Example: Lets say `getMhrRegistrationHomeOwnerGroups.value` isn't initialized as an empty array `[]` or is somehow made `null` or `undefined`. 
Typescript could then cease running this expression (type error). 

In the past, we would use a 'short circuit' approach: ie `getMhrRegistrationHomeOwnerGroups.value && getMhrRegistrationHomeOwnerGroups.value.length...` as a means of protecting against that type error but optional chaining is just less code/neater etc. 

Also, `.length` maybe already handle `null` or `undefined`, this may be more important when accessing object properties or methods etc

Maybe you already know this, sorry if so :) 
Just wanted to share!

Reply:
Yes I am familiar with optional chaining operator, but its good reminder, I tend to forget to use it sometimes 😄